bin/nanocl/src/commands/stats.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+  config::CliConfig,
+  models::{ProcessStatsRow, StatsArg},
+  utils,
+};
+use futures::channel::mpsc;
+use futures::StreamExt;
+use nanocl_error::io::IoResult;
+use std::collections::HashMap;
+
+pub async fn exec_stats(
```

Comment:
Add a test pls, they are in the main.rs file

Reply:
Done